schemas/com.snowplowanalytics.snowplow.badrows/recovery_error/jsonschema/1-0-0

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for errors reported by Snowplow Event Recovery",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.badrows",
+    "name": "recovery_error",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "payload": {
+        "anyOf": [
+            {"$ref": "http://iglucentral.com/schemas/com.snowplowanalytics.snowplow.badrows/adapter_failures/jsonschema/1-0-0#"},
```

Comment:
Don't forget to add new schemas from https://github.com/snowplow/iglu-central/pull/1046